pkgs/development/python-modules/python-docx/default.nix

```
@@ -47,6 +47,16 @@ buildPythonPackage rec {
     "docx"
   ];
 
+  disabledTests = [
+    # lxml.etree.XMLSyntaxError: Blank needed here, line 1, column 21
```

```suggestion
    # https://github.com/python-openxml/python-docx/issues/1302
```
Otherwise it suggests we're just disabling any test that fails.

---

Thanks for the link.